### PR TITLE
Added PyCon US, Updated EuroSciPy and EuroPython

### PR DESCRIPTION
**Pull request by kashfifahim**

### CODE REVIEW

File: 2023.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Subject,Start Date,End Date,Location,Country,Venue,Tutorial Deadline,Talk Deadline,Website URL,Proposal URL,Sponsorship URL
+Subject,Start Date,2023-04-27,Salt,Country,Venue,Tutorial Deadline,Talk Deadline,Website URL,Proposal URL,Sponsorship URL
```
Comment (Mariatta) [inline 2023.csv:1]: This change should be reverted :)

File: 2023.csv [hunk 2]
```diff
@@ -7,5 +7,6 @@ PyCon DE & PyData Berlin 2023,2023-04-17,2023-04-19,"Berlin, Germany",GER,bcc Be
 PyCon LT,2023-05-17,2023-05-20,"Vilnius, Lithuania",LTU,"Vilnius University of Social sciences",,,https://pycon.lt/2023,,
 PyCon Italia 2023,2023-05-25,2023-05-28,"Florence, Italy",ITA,Grand Hotel Mediterraneo,2023-01-15,2023-01-15,https://pycon.it/en,https://pycon.it/cfp,
 DjangoCon EU,2023-05-29,2023-06-02,"Edinburgh, Scotland",GB-SCT,The Assembly Rooms,2023-02-27,2023-02-27,https://2023.djangocon.eu/,https://2023.djangocon.eu/call-for-proposals/,
-EuroPython,2023-07-17,2023-07-23,,,,,,https://europython.eu,,
-EuroSciPy,2023-08-14,2023-08-18,"Basel, Switzerlan",CHE,"Kollegienhaus, University of Basel",,,https://www.euroscipy.org,,
+EuroPython,2023-07-17,2023-07-23,"Prague, Czech Republic",CZE,The Prague Congress Centre,,2023-03-19,https://europython.eu,https://ep2023.europython.eu/cfp,https://ep2023.europython.eu/sponsor/information
+EuroSciPy,2023-08-14,2023-08-18,"Basel, Switzerlan",CHE,"Kollegienhaus, University of Basel",,,https://www.euroscipy.org,https://www.euroscipy.org/2023/program.html,https://www.euroscipy.org/2023/sponsors.html
```
Comment (Mariatta) [inline 2023.csv:11]:
```suggestion
EuroSciPy,2023-08-14,2023-08-18,"Basel, Switzerland",CHE,"Kollegienhaus, University of Basel",,,https://www.euroscipy.org,https://www.euroscipy.org/2023/program.html,https://www.euroscipy.org/2023/sponsors.html
```

**Review by kashfifahim — COMMENTED** [review]
First PR ever, sorry for the silly mistake, lessons learned.

File: 2023.csv [hunk 3]
```diff
@@ -7,5 +7,6 @@ PyCon DE & PyData Berlin 2023,2023-04-17,2023-04-19,"Berlin, Germany",GER,bcc Be
...
 PyCon Italia 2023,2023-05-25,2023-05-28,"Florence, Italy",ITA,Grand Hotel Mediterraneo,2023-01-15,2023-01-15,https://pycon.it/en,https://pycon.it/cfp,
 DjangoCon EU,2023-05-29,2023-06-02,"Edinburgh, Scotland",GB-SCT,The Assembly Rooms,2023-02-27,2023-02-27,https://2023.djangocon.eu/,https://2023.djangocon.eu/call-for-proposals/,
-EuroPython,2023-07-17,2023-07-23,,,,,,https://europython.eu,,
-EuroSciPy,2023-08-14,2023-08-18,"Basel, Switzerlan",CHE,"Kollegienhaus, University of Basel",,,https://www.euroscipy.org,,
+EuroPython,2023-07-17,2023-07-23,"Prague, Czech Republic",CZE,The Prague Congress Centre,,2023-03-19,https://europython.eu,https://ep2023.europython.eu/cfp,https://ep2023.europython.eu/sponsor/information
+EuroSciPy,2023-08-14,2023-08-18,"Basel, Switzerland",CHE,"Kollegienhaus, University of Basel",,,https://www.euroscipy.org,https://www.euroscipy.org/2023/program.html,https://www.euroscipy.org/2023/sponsors.html
+PyCon US,2023-04-19,2023-04-27,"Salt Lake City, Utah",USA,Salt Palace Convention Center,2022-12-11,2022-12-11,https://us.pycon.org/2023/,https://us.pycon.org/2023/speaking/guidelines/,https://us.pycon.org/2023/sponsorship/why-sponsor/
```
Comment (Mariatta) [inline 2023.csv:12]:
```suggestion
PyCon US,2023-04-19,2023-04-27,"Salt Lake City, Utah",USA,Salt Palace Convention Center,2022-12-11,2022-12-11,https://us.pycon.org/2023/,https://us.pycon.org/2023/speaking/guidelines/,https://us.pycon.org/2023/sponsorship/why-sponsor/

```

Add a new line at the end.

**Comment by darjeeling** [comment]
@kashfifahim can you make an update?

**Pull request merged**